recipe/meta.yaml

```diff
@@ -64,13 +64,13 @@ requirements:
     - {{ cdt('libxrender-devel') }}      # [linux]
   host:
     - gobject-introspection 1.*
-    - glib 2.*
-    - gettext   # [osx]
+    - glib {{ glib }}
+    - gettext 0.21.0 # [osx]
     - harfbuzz 4.3.0
```

Comment:
a lot of these deps are specified in the root cbc.yaml, please use `{{ harfbuzz }}` etc (or nothing, although I prefer to be explicit)

Reply:
Fixed